### DISCUSSION TITLE
Lesser-known features of golem?

### DISCUSSION THREAD

**Discussion by rpodcast** [ThinkR-open/golem#568] [General]
I was silly and put this as a reply comment when it belonged in its own post 🤦 .  In any event: 

Greetings from a very happy {golem} user 😀. I'm going to start sharing more of my tips and tricks from my year's journey of releasing 3 major Shiny apps to production that used {golem} but I am curious if there are some lesser known features in the package that people would like to learn more about?

**Comment by ColinFay** [comment]
I'm pretty one of the cool feature in the upcoming release will be how to extend {golem} with templates and hooks! 

https://github.com/ThinkR-open/golem/blob/master/vignettes/f_extending_golem.Rmd

**Reply by rpodcast** [reply]
Oh this looks really neat!  I'll give this a spin when I bootstrap a new application very soon.
